### PR TITLE
[Enhancement] Large incidents widgets - usability changes

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/29128
fixes: https://github.com/demisto/etc/issues/29235

## Description
* Fixed an issue where the widgets/scripts did not work for "All times".
* Changed header Size->Size(MB)
* Changed header Size value to float
* Changed investigation IDs to Hyperlink
* Changed task IDs to Heyperlink
* GetLargestInvestigations now returns by default Text, but can also return table
* GetLargestInputsAndOuputsInIncidents now returns Text result
* Added handling for **playground** investigation
## Does it break backward compatibility?
   - [x] Yes
       - Further details: changes from table to text result (to support links) may adversely affect widgets/scripts that used it before

## ScreenShots
image

image

## Must have
- [x] Tests

@kirbles19 please review:
* Packs/CommonWidgets/Scripts/GetLargestInputsAndOuputsInIncidents/GetLargestInputsAndOuputsInIncidents.yml
* Packs/CommonWidgets/ReleaseNotes/1_0_4.md

### CODE REVIEW

**Comment by DeanArbel** [comment]
@kirbles19 please review:
* Packs/CommonWidgets/Scripts/GetLargestInputsAndOuputsInIncidents/GetLargestInputsAndOuputsInIncidents.yml
* Packs/CommonWidgets/ReleaseNotes/1_0_4.md

File: Packs/CommonWidgets/ReleaseNotes/1_0_4.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#### Widgets
+##### Largest Incidents by Storage Size
+- Removed the MB suffix from the table header **Size**.
+- Changed table header name **Size**->**Size(MB)**.
+- Changed table header **IncidentID** to a hyperlink to the incident.
+##### Largest Inputs And Outputs In Incidents
+- Removed the MB suffix from the table header **Size**.
+- Changed table header name **Size**->**Size(MB)**.
+- Changed table header **IncidentID** to a hyperlink to the incident.
+- Changed table header **TaskID** to a hyperlink to the task.
```
Comment (yuvalbenshalom) [inline Packs/CommonWidgets/ReleaseNotes/1_0_4.md:27]: maybe we should make it less verbose. @kirbles19 WDYT?

**Review by yuvalbenshalom — APPROVED** [review]
nice

File: Packs/CommonWidgets/ReleaseNotes/1_0_4.md (hunk 1 above)
Comment (kirbles19) [reply Packs/CommonWidgets/ReleaseNotes/1_0_4.md:27]: I'll work on these.

File: Packs/CommonWidgets/Scripts/GetLargestInvestigations/GetLargestInvestigations.yml [hunk 2]
```diff
@@ -1,16 +1,33 @@
...
+    you specify 2020-08-14, incidents created on 2020-08-01 will be returned.
   isArray: false
   name: from
   required: false
   secret: false
+- auto: PREDEFINED
+  default: false
+  defaultValue: 'false'
+  description: Change to true to return a result suitable for a table widget. By default
+    the return will be in Markdown.
```
Comment (kirbles19) [inline Packs/CommonWidgets/Scripts/GetLargestInvestigations/GetLargestInvestigations.yml:23]: If "true" results are suitable for a table widget. If "false", results are returned in Markdown. Default is "false".

**Review by kirbles19 — APPROVED** [review]

**Comment by kirbles19** [comment]
@DeanArbel Done.

**Pull request merged**